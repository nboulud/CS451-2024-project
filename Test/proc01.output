d 3 9
d 3 10
d 3 4
d 3 2
d 3 1
d 2 9
d 2 3
d 2 2
d 3 3
d 2 10
d 2 6
d 2 1
d 2 5
d 2 4
d 3 8
d 2 8
d 3 5
d 2 7
d 3 7
d 3 6
