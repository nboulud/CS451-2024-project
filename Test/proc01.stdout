passe par main
Message reçu de 3contenant  9
Ack 9 envoyé à 3
Message reçu de 3contenant  10
Ack 10 envoyé à 3
Message reçu de 3contenant  4
Ack 4 envoyé à 3
Message reçu de 3contenant  2
Ack 2 envoyé à 3
Message reçu de 3contenant  1
Ack 1 envoyé à 3
Message reçu de 2contenant  9
Ack 9 envoyé à 2
Message reçu de 2contenant  3
Ack 3 envoyé à 2
Message reçu de 2contenant  2
Ack 2 envoyé à 2
Message reçu de 3contenant  10
Message reçu de 3contenant  3
Ack 3 envoyé à 3
Message reçu de 2contenant  10
Ack 10 envoyé à 2
Message reçu de 2contenant  6
Ack 6 envoyé à 2
Message reçu de 2contenant  1
Ack 1 envoyé à 2
Message reçu de 2contenant  5
Ack 5 envoyé à 2
Message reçu de 2contenant  4
Ack 4 envoyé à 2
Message reçu de 3contenant  8
Ack 8 envoyé à 3
Message reçu de 3contenant  9
Message reçu de 2contenant  8
Ack 8 envoyé à 2
Message reçu de 3contenant  5
Ack 5 envoyé à 3
Message reçu de 2contenant  7
Ack 7 envoyé à 2
Message reçu de 3contenant  5
Message reçu de 3contenant  7
Ack 7 envoyé à 3
Message reçu de 3contenant  7
Message reçu de 3contenant  2
Message reçu de 3contenant  8
Message reçu de 3contenant  4
Message reçu de 3contenant  5
Message reçu de 3contenant  8
Message reçu de 3contenant  6
Ack 6 envoyé à 3
Message reçu de 3contenant  2
Message reçu de 3contenant  1
Message reçu de 3contenant  7
Message reçu de 3contenant  1
Message reçu de 3contenant  6
Message reçu de 3contenant  4
Message reçu de 3contenant  10
Message reçu de 3contenant  3
Message reçu de 3contenant  3
Message reçu de 3contenant  9
Message reçu de 2contenant  9
Message reçu de 2contenant  10
Message reçu de 2contenant  4
Message reçu de 2contenant  2
Message reçu de 2contenant  8
Message reçu de 2contenant  6
Message reçu de 2contenant  1
Message reçu de 2contenant  5
Message reçu de 2contenant  3
Message reçu de 2contenant  3
Message reçu de 2contenant  1
Message reçu de 3contenant  3
Message reçu de 3contenant  4
Message reçu de 2contenant  7
Message reçu de 2contenant  4
Message reçu de 2contenant  7
Message reçu de 2contenant  9
Message reçu de 2contenant  10
Message reçu de 3contenant  2
Message reçu de 3contenant  1
Message reçu de 3contenant  6
Message reçu de 3contenant  8
Message reçu de 2contenant  6
Message reçu de 3contenant  7
Message reçu de 2contenant  2
Message reçu de 3contenant  8
Message reçu de 3contenant  10
Message reçu de 2contenant  8
Message reçu de 3contenant  4
Message reçu de 3contenant  5
Message reçu de 3contenant  2
Message reçu de 2contenant  5
Message reçu de 3contenant  6
Message reçu de 2contenant  5
Message reçu de 2contenant  4
Message reçu de 2contenant  9
Message reçu de 2contenant  4
Message reçu de 2contenant  6
Message reçu de 2contenant  1
Message reçu de 3contenant  5
Message reçu de 2contenant  4
Immediately stopping network packet processing.
Writing output.
